### PR TITLE
MCR-2378 (1) remove guice dependency (MCRProcessableRegistry)

### PR DESCRIPTION
**Pull request by rsteph-de**
[Link to jira](https://mycore.atlassian.net/browse/MCR-2378).

### CODE REVIEW

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on ebe30f7cff7f74fa68339fe3e9045d9a48d1bed0! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/6880-pull_request-1219

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on e3d649f29155224af6b4213f8cd5826323ff011e! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/6882-pull_request-1219

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 0db161528a294418978ee9e12e47188c2b9b42d8! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/6884-pull_request-1219

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on e3d649f29155224af6b4213f8cd5826323ff011e! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/6882-pull_request-1219

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 6ccbf89b4de3f48d68acd22c5f6b0459ed6d48f5! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/6886-pull_request-1219

File: mycore-base/src/main/java/org/mycore/frontend/fileupload/MCRUploadHandlerManager.java [hunk 1]
```diff
@@ -48,7 +48,8 @@
     static {
         HANDLERS = new MCRCache<>(100, "UploadHandlerManager UploadHandlers");
         COLLECTION = new MCRProcessableDefaultCollection("Upload Manager");
-        MCRProcessableRegistry registry = MCRInjectorConfig.injector().getInstance(MCRProcessableRegistry.class);
+        MCRProcessableRegistry registry = MCRConfiguration2
+            .<MCRProcessableRegistry>getSingleInstanceOf("MCR.Processable.Registry.Class").orElseThrow();
```
Comment (sebhofmann) [inline mycore-base/src/main/java/org/mycore/frontend/fileupload/MCRUploadHandlerManager.java]: Move this to new Util Class e.G. MCRProcessableRegistryHelper.getInstance()

**Review by sebhofmann — CHANGES_REQUESTED** [review]

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 24c0c5eb1561a59235c807826dd748f19f4e0733! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/6911-pull_request-1219

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 96965675b89bbe994244db586cc36913b8965aad! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/6917-pull_request-1219

File: mycore-base/src/main/java/org/mycore/frontend/fileupload/MCRUploadHandlerManager.java (hunk 1 above)
Comment (rsteph-de) [reply mycore-base/src/main/java/org/mycore/frontend/fileupload/MCRUploadHandlerManager.java]: solved by static method in interface MCRProcessableRegistry

**Review by rsteph-de — COMMENTED** [review]
done

**Review by sebhofmann — APPROVED** [review]

**Pull request merged**